hotsos/core/ycheck/events.py

```diff
@@ -385,6 +623,10 @@ def run(self, results):
                 if not ret:
                     continue
 
+                log.debug("event %s.%s callback '%s' return size is %d bytes",
+                          event_results_obj.section, event, callback_name,
+                          sys.getsizeof(ret))
```

Comment:
`sys.getsizeof()` could be misleading for nested lists/dicts/etc. as it does not follow references.

```python
Python 3.12.3 (main, Apr 10 2024, 05:33:47) [GCC 13.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import sys
>>> class b:
...     def __init__(self):
...             self.dict = {'a':5}
... 
>>> obj = b()
>>> sys.getsizeof(obj)
48
>>> class c:
...     def __init__(self):
...             pass
... 
>>> obj2 = c()
>>> sys.getsizeof(obj)
48
>>> sys.getsizeof(obj2)
48
>>> obj.dict = {'a':5, 'b':10, 'c':12}
>>> sys.getsizeof(obj)
48
>>> sys.getsizeof(obj.dict)
184

Reply:
yeah it is in act not giving me the information i was hoping

Reply:
i have gone and removed this now